GTG/backends/__init__.py

```diff
@@ -32,7 +32,7 @@
 from GTG.backends.generic_backend import GenericBackend
 
 log = logging.getLogger(__name__)
-
+inactive_modules = set()
```

Comment:
Might be better to move this to the `BackendFactory` class below. The [`Borg`](https://github.com/getting-things-gnome/gtg/blob/5f72c8a9319e06d42967fb8461ea832e8cd095d9/GTG/core/borg.py) class it inherits is basically making it a singleton (state shared on every instance).